vaadin-archetype-application/src/main/resources/archetype-resources/pom.xml

```diff
@@ -112,7 +92,7 @@
             <plugin>
                 <groupId>org.eclipse.jetty</groupId>
                 <artifactId>jetty-maven-plugin</artifactId>
-                <version>9.4.15.v20190215</version>
+                <version>9.4.36.v20210114</version>
                 <configuration>
                     <scanIntervalSeconds>1</scanIntervalSeconds>
```

Comment:
Since I can't make a multiline suggestion outside your changes (???), please remove this scanIntervalSeconds as it will not work reliably with live reload and just cause problems, lets ask people to use devcm instead.
Or I can make another PR to fix it